spec/definitions/CheckoutPage.yaml

```diff
@@ -128,6 +128,18 @@ properties:
     uniqueItems: true
     items:
       $ref: "#/definitions/FieldConfig"
+  taxes:
+    type: object
+    description: Taxes, you can specify description and list of name-value items.
+    properties:
+      description:
+        description: Taxes description.
+        type: string
+      taxes:
```

Comment:
I think it's better to rename to `items` to avoid same nested item name as parent item name (tautology).

@adamaltman What you think about that?